frontend/src/components/SampleQueries.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Glyphicon } from 'react-bootstrap';
+import './SampleQueries.less';
+
+const queries = [
+  {
+    name: 'load all java files',
```

Comment:
I'd avoid "todos" if possible:
- I'd rename first example as "get all commits", what matches this sql.
- And next examples I'd make them self explaining: removing them if they should be provided by business, or use simple ones like "get all repositories" and "get uast from code"